ribbon-loadbalancer/src/main/java/com/netflix/loadbalancer/BaseLoadBalancer.java

```diff
@@ -160,10 +160,10 @@ public BaseLoadBalancer(IClientConfig config) {
     }
 
     public BaseLoadBalancer(IClientConfig config, IRule rule, IPing ping) {
-        initWithConfig(config, rule, ping);
+        initWithConfig(config, rule, ping, createLoadBalancerStatsFromConfig(config));
     }
     
-    void initWithConfig(IClientConfig clientConfig, IRule rule, IPing ping) {
+    void initWithConfig(IClientConfig clientConfig, IRule rule, IPing ping, LoadBalancerStats stats) {

```

Comment:
nit: this is package private so 'should' be safe to do, but just in case since Ribbon is such an old library, can you do an additional overload method to be compatible? Thanks.
```
void initWithConfig(IClientConfig clientConfig, IRule rule, IPing ping) {
    initWithConfig(IClientConfig clientConfig, IRule rule, IPing ping, createLoadBalancerStatsFromConfig(config));
}
```